apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -63,21 +63,38 @@ This project comes with TypeScript configured and has the following structure.
 
 ### Configuring the project to use PostgreSQL
 
+Prisma init migrations will try to drop `postgres` database by default, and this may lead to conflicts with Supabase initial migrations that are critically needed for the correct functioning of our managed databases. For such cases Prisma has a feature called [Shadow Databases](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).
```

Comment:
The first sentence is a bit confusing so my suggestion here might not have the same meaning you were trying to get across.
```suggestion
By default, Prisma migrations will try to drop the `postgres` database, which can lead to conflicts with Supabase databases. For this scenario, use [Prisma Shadow Databases](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).
```